packages/compass-indexes/src/components/indexes-toolbar/indexes-toolbar.tsx

```diff
@@ -132,6 +150,43 @@ export const IndexesToolbar: React.FunctionComponent<IndexesToolbarProps> = ({
                 signals={PerformanceSignals.get('too-many-indexes')}
               />
             )}
+            {isSearchManagementActive && (
+              <SegmentedControl
+                onChange={onChangeIndexesSegment}
+                className={alignSelfEnd}
+                label="Viewing"
+                defaultValue="regular-indexes"
+              >
+                <SegmentedControlOption value="regular-indexes">
+                  Indexes
+                </SegmentedControlOption>
+                {!isAtlasSearchSupported && (
+                  <Tooltip
```

Comment:
Tooltip is necessary for showing the tooltip (aha) when it's disabled. However, it breaks the SegmenedControlOption, and clicks don't work.

That is why I'm branching here and we only add the Tooltip when we know it's disabled, as clicking doesn't matter.